.env

```diff
@@ -7,6 +7,9 @@ REACT_APP_BUILD_VERSION=
 REACT_APP_API=https://nexus.oasis.io/v1/
 REACT_APP_TESTNET_API=https://testnet.nexus.oasis.io/v1/
 REACT_APP_TITLE=Oasis Explorer
+REACT_APP_IMAGE=oasis-og-image.jpg
```

Comment:
Can we add `_META_` into env name, so that it is more clear what is it doing.

Reply:
> Can we add `_META_` into env name, so that it is more clear what is it doing. And for similar env variables as well.

Done.